extensions/amp-story/1.0/amp-story.js

```diff
@@ -164,6 +164,20 @@ const MINIMUM_AD_MEDIA_ELEMENTS = 2;
  */
 const STORY_LOADED_CLASS_NAME = 'i-amphtml-story-loaded';
 
+/**
+ * CSS class for the opacity layer that separates the amp-sidebar and the rest
+ * of the story when the amp-sidebar is entering the screen.
+ * @const {string}
+ */
+const OPACITY_MASK_CLASS_NAME = 'i-amphtml-story-opacity-mask';
+
+/**
+ * CSS class for the opacity layer that separates the amp-sidebar and the rest
+ * of the story when the amp-sidebar is leaving the screen
+ * @const {string}
+ */
+const OPACITY_MASK_HIDDEN_CLASS_NAME = 'i-amphtml-story-opacity-mask-hidden';
```

Comment:
Can we add a `[hidden]` attribute on `OPACITY_MASK_CLASS_NAME` instead, so we don't have to declare two classes?
`.i-amphtml-story-opacity-mask[hidden] {/* opacity + transition + pointer-events overrides */}`